src/index.jsx

```diff
@@ -1,11 +1,14 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { UserProvider } from './context/userContext';
 import { App } from './App';
 import { reportWebVitals } from './reportWebVitals';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <UserProvider>
```

Comment:
why are you going with strict mode?

Reply:
https://stackoverflow.com/questions/53183362/what-is-strictmode-in-react

Reply:
https://reactjs.org/docs/strict-mode.html#:~:text=StrictMode%20is%20a%20tool%20for,and%20warnings%20for%20its%20descendants.